x-pack/plugins/security_solution/common/endpoint/constants.ts

```diff
@@ -62,3 +62,6 @@ export const failedFleetActionErrorCode = '424';
 
 export const ENDPOINT_DEFAULT_PAGE = 0;
 export const ENDPOINT_DEFAULT_PAGE_SIZE = 10;
+
+// total number of transforms we should have running for endpoint
+export const ENDPOINT_TRANSFORMS_COUNT = 2;
```

Comment:
Is there a better approach than keeping a static total? This can be problematic in the future if we add more transforms, but don't realize that we need to also adjust this here. Anyway we can (perhaps on Plugin setup/start) pull the package info and get a count out of there?

Reply:
if we're able to fetch the raw-ish package contents from fleet (... can we?), we can then count up the desired transforms. But I don't think that's exposed (or even saved in a nice way anywhere accessible??)

Reply:
The only thing I can think of right now is filtering down transforms using the `paths` returned from `getRegistryPackage`. We would just need to make sure to pass the currently installed package version to `getRegistryPackage` otherwise we might run into issues if an older version is installed and the newer one has a different transform count.